cleanrepo/CleanRepo/Program.cs

```diff
@@ -1570,13 +1678,13 @@ private static List<FileInfo> GetYAMLFiles(string directoryPath)
     /// </summary>
     private static Dictionary<string, List<string>> GetMediaFiles(string mediaDirectory, bool searchRecursively = true)
     {
-        DirectoryInfo dir = new(mediaDirectory);
+        DirectoryInfo dir = new DirectoryInfo(mediaDirectory);
```

Comment:
If you're using .NET 8/C# 12, the previous syntax works. It's a newer, more concise syntax.